src/styles/reset.scss

```diff
@@ -0,0 +1,108 @@
+/* stylelint-disable-next-line no-invalid-double-slash-comments */
+//YUI 3.18.1 (build f7e7bcb)
+//Copyright 2014 Yahoo! Inc. All rights reserved.
+//Licensed under the BSD License.
+//http://yuilibrary.com/license/
+
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+fieldset,
+img {
+  border: 0;
+}
+
+address,
+caption,
+cite,
+code,
+dfn,
+em,
+strong,
+th,
+var {
+  font-style: normal;
+  font-weight: normal;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+caption,
+th {
+  text-align: left;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: 100%;
+  font-weight: inherit;
+}
+
+q::before,
+q::after {
+  content: '';
+}
+
+abbr,
+acronym {
+  border: 0;
+  font-variant: normal;
+}
+
+sup {
+  vertical-align: text-top;
+}
+
+sub {
+  vertical-align: text-bottom;
+}
+
+input,
+textarea,
+select {
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+legend {
+  color: currentcolor;
+}
```

Comment:
A `button` reset [like this](https://www.trysmudford.com/blog/a-good-reset/) would be nice too - although maybe it's fair game to let them write it?

Reply:
Nice, great shout 🚀  And great article 🚀